vnext/Microsoft.ReactNative/IJSValueWriter.idl

```diff
@@ -18,7 +18,7 @@ namespace Microsoft.ReactNative
     "See the @IJSValueReader for the corresponding reader interface.\n"
     "\n"
     "The @IJSValueReader and @IJSValueWriter must be rarely used directly. "
-    "Use them to create serializer and deserializer functions for a type. "
+    "Use them to create functions that serialize and deserialize from/to a native type. "
```

Comment:
>from/to [](start = 65, length = 7)

Should we reverse it to "to/from"? "serialize to" and "deserialize from". Then, the next question is if we must swap the IJSValueWriter and IJSValueReader above to match the IJSValueWriter to serialization and IJSValueReader to deserialization.